kafka/consumer/group.py

```diff
@@ -462,7 +462,7 @@ def _poll_once(self, timeout_ms):
             return records
 
         self._fetcher.init_fetches()
-        self._client.poll(timeout_ms)
+        self._client.poll(timeout_ms, sleep=True)
```

Comment:
This is a substantial change - why? The current consumer design expects poll() to return immediately if there is nothing to block on.